appserver/node-express/lib/db-client/qnaDoc/patch.js

```diff
@@ -66,6 +116,24 @@ module.exports = function (txid, spec) {
     );
   };
 
+  /**
+   * Handle the post of a new answer to a Samplestack question. The posted
+   * answer is merged with an answer template object, a unique ID, contributor
+   * information, and the current time. The resulting object is written to the
+   * MarkLogic database. During the write, the answer count for the question is
+   * incremented and the last activity date is updated.
+   *
+   * @param  {String} txid The transaction ID.
+   * @param  {Object} contributor Contributor object. Example:
+   *   {"id":"cf99542d-f024-4478-a6dc-7e723a51b040",
+   *    "displayName":"JoeUser"}
+   * @param  {String} questionId The question ID.
+   * @param  {Object} spec The answer content from the request. Example:
+   *   {"comments":[],
```

Comment:
I figured out why the empty comments array is there when it seems like it doesn't need to be. I'd forgotten how it all works in the browser tier. We create a "draft" answer for our answer form on the QnADoc page. That draft answer has an empty comments array (as part of the standard ssAnswer creation) even though there's nothing that would be put in the comment array from the new answer form. (The new answer form has fields for just the title and the body text.) 

We can delete that empty comments array for the draft answer (after its creation) or we can leave the array there without affecting how the app works to the end user. But considering that Dave's students will be working with the new answer functionality, removing the comments array would make it slightly less confusing IMO. And removing that comments array means we wouldn't have to explain (in our code comments) why that empty array is there. @laurelnaiad any thoughts?